### PR TITLE
fix: droppable-get-dimension withScroll

### PR DESCRIPTION
**Pull request by woohling**
We're forking your amazing repo and need to add some features based on our product logic, and then I found a potential bug, which I spent a lot of time debuging.

The default container supporting horizontal scroll is the window, which would not trigger any bugs currently. But I think it would be stricter to code it this way. Pls take a look and see if I'm doing this correctly.

### CODE REVIEW

**Comment by woohling** [comment]
@alexreardon kindly pls take a look at the pull request

**Comment by alexreardon** [comment]
Away this week. Will look next week

File: src/view/droppable-dimension-publisher/get-dimension.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ export default ({
 
     return {
       client: frameClient,
-      page: withScroll(frameClient),
```
Comment (alexreardon) [inline src/view/droppable-dimension-publisher/get-dimension.js]: `withScroll` will apply the page scroll

Comment (alexreardon) [reply src/view/droppable-dimension-publisher/get-dimension.js]: You are right that your change is a bit stricter in that it does not read the page scroll directly, but just supplies it..

Thoughts?

Comment (alexreardon) [reply src/view/droppable-dimension-publisher/get-dimension.js]: I also like that this approach does not touch the DOM again and just uses the value that has been passed in

**Pull request merged**